### PR TITLE
Made background color persistent

### PR DESCRIPTION
**Pull request by BilalAlam173**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -33,9 +33,7 @@ function createAvatar(el) {
 
 	name = el.getAttribute("data-avatar");
 	if(name) {
-		const idx1 = Math.abs(name[0].charCodeAt() - 64);
-		const idx2 = name.length > 1 ? Math.abs(name[1].charCodeAt() - 64) : 0;
-		const idx_from_alphabets = ((idx1 * (idx2)) + idx1);
+		const idx = name.split('').map(x=> x.charCodeAt(0).toString()).join('') & colors.length
```
Comment (nopantsmonkey) [inline script.js]: add space before => symbol

Comment (nopantsmonkey) [reply script.js]: please simplify the idx calculation.

**Comment by BilalAlam173** [comment]
all done except => simplify idx calculation

File: script.js [hunk 2]
```diff
@@ -32,14 +32,14 @@ function createAvatar(el) {
 	}
 
 	name = el.getAttribute("data-avatar");
-	if(name) {
-		const idx = name.split('').map(x=> x.charCodeAt(0).toString()).join('') & colors.length
-		el.setAttribute("style", "background-color:" + colors[idx_from_alphabets % colors.length]);
+	if (name) {
+		const idx = name.split('').map(x => x.charCodeAt(0).toString()).join('') % colors.length
+		el.setAttribute("style", "background-color:" + colors[idx % colors.length]);
```
Comment (nopantsmonkey) [inline script.js]: i think you dont need '% colors.length' here, you are calculating that in the idx variable above

**Comment by BilalAlam173** [comment]
done

**Comment by nopantsmonkey** [comment]
Please commit the bundle too.

**Comment by BilalAlam173** [comment]
> Please check your code with the demo.html file and then commit the bundle too.

done

**Pull request merged**